pyproject.toml

```diff
@@ -0,0 +1,8 @@
+[build-system]
```

Comment:
btw, by adding this you effectively break editable mode.

Reply:
I defer, I tested both `pip install .` and `pip install -e .` and they both worked fine (py37/macos). Did you tried them and failed? If so, which environment?

Reply:
The problem is that PEP517 has no notion of "editable install" atm.

Reply:
pip==19.3

Reply:
@webknjaz Also check https://github.com/ansible/ansible-lint/pull/592 which is supposed to fix the setup.py -- I do not care which one goes in as long we fix ansible-lint itself. It is a huge pain to introduce workarounds everywhere is used.

Reply:
Both break things, hence not ready for merge.

Reply:
> Both break things, hence not ready for merge.

That sentence is a bit blurry and clearly not actionable. What use-case is broken? What needs to be done to bring it to ready-to-merge status?

Reply:
PEP517 ignores the existence of editable installs making Pip behavior unpredictable. I know of at least one Pip version that errors out under such circumstances.

Reply:
The action here is to wait for the decision whether we can drop support for older envs.